numpy/linalg/linalg.py

```diff
@@ -1334,48 +1343,77 @@ def svd(a, full_matrices=True, compute_uv=True):
     Notes
     -----
 
-    .. versionadded:: 1.8.0
-
-    Broadcasting rules apply, see the `numpy.linalg` documentation for
-    details.
-
-    The decomposition is performed using LAPACK routine _gesdd
+    The decomposition is performed using LAPACK routine ``_gesdd``.
 
-    The SVD is commonly written as ``a = U S V.H``.  The `v` returned
-    by this function is ``V.H`` and ``u = U``.
+    SVD is usually described for the factorization of a 2D matrix :math:`A`.
```

Comment:
Here I tried
```
    SVD is usually described for the factorization of a 2D matrix :math:`A`.
    The higher-dimensional case will be discussed below. In the 2D case, SVD is
    written as :math:`A = U S V^H`, where :math:`A = a`, :math:`U= u`,
    :math:`S= \mathtt{np.diag}(s)` and :math:`V^H = vh`. The 1D array `s` contains
    the singular values of `a` and `u` and `vh` are unitary. The rows of `vh`
    are the eigenvectors of :math:`A^H A` and the columns of `u` are the
    eigenvectors of :math:`A A^H`. In both cases the corresponding (possibly
    non-zero) eigenvalues are given by ``s**2``.
```
which I think looks a touch better and also avoids the potential to break across lines.

Reply:
Note that some of the lines are also >= 80 characters, they should be < 80.

Reply:
ok